components/CommentForm.vue

```diff
@@ -11,27 +11,41 @@
 </template>
 
 <script lang="ts">
-import { Vue, Component, Prop } from 'nuxt-property-decorator'
-import { notesStore } from '@/store'
+import { defineComponent, reactive } from '@nuxtjs/composition-api'
+import { useNoteStore } from '@/composables/use-note'
+import { useAuthStore } from '@/composables/use-auth'
 
-@Component
-class CommentForm extends Vue {
-  @Prop({ default: false })
-  noteId!: string
-
-  private commentContent: string = ''
+export default defineComponent({
+  props: {
+    noteId: {
+      type: String,
+      require: true,
+    },
+  },
+  setup(props) {
+    const state = reactive({
+      comment: '',
+    })
+    const content = () => {
+      return state.comment
+    }
+    const { createComment } = useNoteStore()
+    const { user, isValid } = useAuthStore()
 
-  get content() {
-    return this.commentContent
-  }
+    if (!isValid) {
```

Comment:
useAuthStoreの返り値のisValidはリアクティブではない？
チラ見したらcomputedが使われているっぽいから、↓みたいに書かないと駄目じゃない？

```suggestion
    if (!isValid.value) {
```

そしてそれだと最初の一回しか実行されないから意図した挙動にならなそう。

Reply:
この（.valueを書かない）やりがちなミスもeslintで防げます。

https://www.sonicgarden.world/groups/3620/entries/1023087

Reply:
https://www.yurikago-blog.com/articles/19

記事を参考に、`eslint.js`のrulesに設定するとおもったのですが、設定がきかない。何か設定もれしているのでしょうか？

```
parser: "vue-eslint-parser",
parserOptions: {
  "parser": "@typescript-eslint/parser"
},
rules: {
  'strict-boolean-expressions' : true
}
```

Reply:
https://github.com/typescript-eslint/typescript-eslint/tree/master/packages/eslint-plugin#supported-rules

Reply:
https://palantir.github.io/tslint/rules/strict-boolean-expressions/

Reply:
@konti-kun がnuxt環境で有効にしていた気がするから聞いてみるといいかも？

ちなみに設定が効かないだけだと何をどこまで動作確認したのか分からないので、もうちょっとどういう方法でどこまで確認したのか書いておくとアドバイスもらいやすいかもです。